src/main/java/org/junit/Assert.java

```diff
@@ -812,6 +812,37 @@ public static void assertNotSame(Object unexpected, Object actual) {
         assertNotSame(null, unexpected, actual);
     }
 
+    /**
+     * Asserts that two {@link CharSequence} instances have the same contentsa.
+     * If they do not, an {@link AssertionError} is thrown.
+     *
+     * @param expected the expected contents
+     * @param actual the object to compare to
+     */
+    public static void assertContentEquals(CharSequence expected, CharSequence actual)  {
```

Comment:
I'm wondering if the expected value should be a String (the test author would usually pass in a literal string or a constant of type `Strnig`). What do you think?